src/components/picker/index.tsx

```diff
@@ -326,7 +295,7 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
               value={label}
               selection={Constants.isAndroid ? {start: 0} : undefined}
             >
-              {renderPickerInnerInput()}
+              {renderPickerInnerInput}
```

Comment:
Looks good. Maybe we should rename this because its not a render function anymore? 
Approved anyway.